common/migrations/0059_import_bmj_journals.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import json
+import io
+
+from django.db import migrations
+from common.models import Journal
+
+import logging
+logger = logging.getLogger('django')
+
+BMJ_JOURNALS = 'cos/static/BMJJournals.json'
+
+
+def add_BMJ_journals(*args, **kwargs):
+
+    BMJ_journals = json.load(io.open(BMJ_JOURNALS, 'r', encoding='utf-8-sig'))
+
+    for entry in BMJ_journals:
+        journal, created = Journal.objects.get_or_create(
```

Comment:
Just an FYI (not a blocker), you can actually do this all at once using the [`defaults` argument](https://docs.djangoproject.com/en/1.11/ref/models/querysets/#get-or-create):

```
journal, created = Journal.objects.get_or_create(
	title=entry['Journal'],
	publisher='BMJ',
	defaults={
        is_preregistered_journal = entry['Is Preregistered?']
        is_top_journal = entry['IS TOP?']
        area = entry['Discipline']
	}
)
```